project/Build.scala

```diff
@@ -1144,9 +1163,20 @@ object Build extends sbt.Build {
             val outFile = dir / "SourceMapTest.scala"
             IO.write(outFile, replaced.replace("0/*<testCount>*/", i.toString))
             Seq(outFile)
-          }
+          },
+
+        testFrameworks += new TestFramework("com.novocode.junit.JUnitFramework"),
+        testOptions += Tests.Argument(
+          new TestFramework("com.novocode.junit.JUnitFramework"), "-v", "-a"),
```

Comment:
I think you can replace `new TestFramework("com.novocode.junit.JUnitFramework")` by `TestFrameworks.JUnit`.